README.md

```diff
@@ -1 +1,18 @@
 # hw01
+## Hello there! 
+
+My name is Julia Pruin and I am a new MAPSS student here at UChicago. I am concentrating in Psychology and working with [Dr. Monica Rosenberg](https://psychology.uchicago.edu/directory/monica-rosenberg) for my thesis. I plan to research the cognitive measures that might be used to predict changing ADHD diagnoses in an adolescent sample. 
+
+## Some Fun Facts
+
+ - I am originally from San Antonio, TX and this is my first time living outside Texas.
+ - I love to bake and cook. If I am stressed, I make cookies or bread.
+ - I am not allergic to mosquitoes. I still get bit, but I don't itch or swell.
+ - I have a pitbull. Her name is Hayley. She is a baby and I miss her very much.
+
+## Some Progress Notes
+ - I really didn't pay enough attention the first time that I learned how to fork from github in June. I have been spoiled by downloading from Box for the last few months and so I had to *entirely* relearn the process.
+ - I was always curious as to how R for Data Science and other R things were made pretty for the web. Now I know that it was just R Markdown. Neat.
+ - After I relearned the forking process it was pretty smooth sailing. I had trouble imbedding a picture so I gave up but I figured out how to use emojis instead :smile:
+ - I felt like this was very similar to ggplot in that I can easily remember the bascis of the grammar but I have to keep checking articles and cheatsheets to use specfic shapes, colors, emphasis, etc.
+ - I found a cheatsheet that I had high hopes for because I keep [a nice one for ggplot](https://rstudio.com/wp-content/uploads/2015/03/ggplot2-cheatsheet.pdf), but [this thing](https://rstudio.com/wp-content/uploads/2016/03/rmarkdown-cheatsheet-2.0.pdf?_ga=2.40532778.94379362.1601512842-120555137.1591728115) is a monster and it makes me sad. :unamused:
```

Comment:
You can try [this cheat sheet](https://rstudio.com/wp-content/uploads/2015/02/rmarkdown-cheatsheet.pdf) instead. It's a little bit more readable